### PR TITLE
WIP: add RHOAI Notebooks Performance tests to the HOME page

### PR DESCRIPTION
**Pull request by kpouget**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This PR adds PSAP RHOAI Notebooks Performance tests to the HOME page of the CPT Dashboard

Note that this PR includes multiple WIP commits. These commits will not be included in the final version of the PR.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by jtaleric** [comment]
Bad bot.

**Comment by jtaleric** [comment]
@kpouget is this still being worked on?

**Comment by kpouget** [comment]
it's on hold, too many things to do, too few persons! 
But the idea is still open, we'll get to it, sooner or later :)

File: backend/app/api/v1/endpoints/cpt/maps/rhoai.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    df.insert(len(df.columns), "product", "RHOAI Notebooks Performance")
+    df.insert(len(df.columns), "ciSystem", "Middleware Jenkins")
+    df.insert(len(df.columns), "releaseStream", "rc")
+    df.insert(len(df.columns), "buildUrl", "N/A")
+    df.insert(len(df.columns), "jobStatus", "success")
+
+    if df.empty:
+        return df
+
+    df["startDate"] = df["metadata.start"]
```
Comment (vishnuchalla) [inline backend/app/api/v1/endpoints/cpt/maps/rhoai.py:20]: can we remove these `metadata.*` columns from the df once they are assigned to new columns as they are not being used further anyway?

File: backend/app/api/v1/endpoints/rhoai/jobs.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    if start_date > end_date:
+        return Response(content=json.dumps({'error': "invalid date format, start_date must be less than end_date"}), status_code=422)
+
+    results = await rhoai_getData(start_date, end_date, 'rhoai.elasticsearch')
+
+    if len(results) >= 1 :
+        response = {
+            'startDate': start_date.__str__(),
+            'endDate': end_date.__str__(),
+            'results': results.to_dict('records')
```
Comment (vishnuchalla) [inline backend/app/api/v1/endpoints/rhoai/jobs.py:43]: Only this field seems to be differing in this if-else block.

File: backend/app/services/search.py [hunk 3]
```diff
@@ -27,6 +29,12 @@ def __init__(self,configpath="",index=""):
                     verify_certs=False,
                     http_auth=(self.esUser,self.esPass)
             )
+            self.sync_es = Elasticsearch(
```
Comment (vishnuchalla) [inline backend/app/services/search.py:32]: might need resolving some conflicts here.

**Comment by kpouget** [comment]
thanks @vishnuchalla for having a loook,
this work is currently on hold because of lack of time, but we'll get back to it :)

**Pull request closed without merging**